meshroom/core/attribute.py

```diff
@@ -143,6 +144,10 @@ def setEnabled(self, v):
         self._enabled = v
         self.enabledChanged.emit()
 
+    def getUidIgnoreValue(self):
+        """ Value for which the attribute should be ignored during the UID computation. """
+        return self._uidIgnoreValue
```

Comment:
`return self.attributeDesc.uidIgnoreValue`

No need to duplicate the value into the node. We can just use the value from the attribute description.

Reply:
Or maybe it's needed when we are in Compatibility mode?

Reply:
It doesn't seem to be needed in Compatibility Mode, so we can indeed avoid duplicating it in the node. Done!